data/json/mapgen/dairy_farm.json

```diff
@@ -0,0 +1,251 @@
+[
+  {
+    "type":"monstergroup",
+    "name" : "GROUP_COWS",
+    "is_safe" : true,
+    "default" : "mon_null",
+    "monsters":[{"monster" : "mon_cow", "cost_multiplier" : 10, "freq": 100, "ends" : 168 },
+    {"monster" : "mon_cow", "freq": 50, "cost_multiplier" : 5, "starts" :  336  }
+    ]
```

Comment:
Theres a period of time where no cows at all will spawn:  between 168 and 336. 
  Im not sure if the quantity means hours or not, but theres a gap nontheless.

additionally, with a cost multiplier that high, its no wonder you wont get any cows out of a passive building.  Later on in the game theres more 'spawn points' available, but early game theres not nearly enough in your average mapgen structure to make a cow.

Try a cost of 1 or 0 for now.  You have already made them only spawn 10% of the time and 5%, respectively.